### PR TITLE
zh: Update hardware-and-software-requirements.md

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added, or deleted? (Required)

some description need refine

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by GMHDBJD** [comment]
LGTM

File: zh/hardware-and-software-requirements.md [hunk 1]
```diff
@@ -49,5 +49,5 @@ DM 支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器
 
 > **注意：**
 >
-> - 在生产环境中，DM-master 和 DM-worker 可以部署和运行在同一个服务器上。
-> - 在生产环境中，推荐使用比如上表格中配置更高的服务器。
+> - 在生产环境中，DM-master 和 DM-worker 不建议部署和运行在同一个服务器上，以防 DM-worker 对磁盘的写入干扰 DM-master 高可用组件使用磁盘。
```
Comment (CharLotteiu) [inline zh/hardware-and-software-requirements.md:52]:
```suggestion
> - 在生产环境中，不建议将 DM-master 和 DM-worker 部署和运行在同一个服务器上，以防 DM-worker 对磁盘的写入干扰 DM-master 高可用组件使用磁盘。
```

File: zh/hardware-and-software-requirements.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ aliases: ['/docs-cn/tidb-data-migration/dev/hardware-and-software-requirements/'
 
 # DM 集群软硬件环境需求
 
-DM 支持主流的 Linux 操作系统环境，可以很好的部署和运行在 Intel 架构服务器环境及主流虚拟化环境。
+DM 支持主流的 Linux 操作系统，可以部署和运行在 Intel 架构服务器环境及主流虚拟化环境中。
```
Comment (CharLotteiu) [inline zh/hardware-and-software-requirements.md:9]:
```suggestion
DM 支持主流的 Linux 操作系统，可以在 Intel 架构服务器环境及主流虚拟化环境中部署和运行。
```

**Review by CharLotteiu — COMMENTED** [review]
rest LGTM

File: zh/hardware-and-software-requirements.md (hunk 2 above)
Comment (IANTHEREAL) [reply zh/hardware-and-software-requirements.md:9]: 这句话貌似不通顺，`DM 支持在主流的 Linux 操作系统，Intel 架构服务器和主流虚拟化环境中部署和运行`，是不是更好

File: zh/hardware-and-software-requirements.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ DM 支持主流的 Linux 操作系统环境，可以很好的部署和运行在
 
 > **注意：**
 >
-> 以上 Linux 操作系统可运行在物理服务器以及 VMware、KVM、XEN 主流虚拟化环境上。
+> 以上 Linux 操作系统可运行在物理服务器以及 VMware、KVM、XEN 主流虚拟化环境上。在虚拟化环境中使用 DM 时，CPU、内存、磁盘的性能可能会产生损耗，从而导致 DM 性能下降。
```
Comment (IANTHEREAL) [inline zh/hardware-and-software-requirements.md:22]: 为什么要特别标注这个。 虚拟化环境性能不高，这个应该是尝试

**Comment by IANTHEREAL** [comment]
LGTM

**Review by CharLotteiu — APPROVED** [review]
LGTM

**Pull request merged**